lang_GENERIC/analyze/CFG_build.ml

```diff
@@ -34,6 +34,8 @@ module F = IL (* to be even more similar to controlflow_build.ml *)
 (* Types *)
 (*****************************************************************************)
 
+type cfg_label = string * G.sid
```

Comment:
maybe a comment like (* like IL.label but without the token attached to the ident *)

Reply:
Done!